PageViews/PageViews.module.less

```diff
@@ -34,11 +34,31 @@
 		height: 132px;
 	}
 
-	.contentsArea, .viewManager{
+	&.full .steps {
+		position: absolute;
+		align-self: center;
+		bottom: 0px;
+		height: 132px;
```

Comment:
Let's consider specifying 'min-height' rather than 'height' so we could remove a unwanted <div> wrapper node.

Reply:
Thank you. I fixed it to use `min-height`